hasher-matcher-actioner/webapp/src/pages/ContentDetails.jsx

```diff
@@ -37,7 +38,8 @@ export default function ContentDetails() {
   useEffect(() => {
     fetchImage(id)
       .then(result => {
-        setImage(URL.createObjectURL(result));
+        console.log(result);
```

Comment:
rm

Reply:
surprised the linter didn't complain, we should probably make the full change over to .tsx as I think that is likely the reason.